lib/KpmManager.py

```diff
@@ -162,7 +162,7 @@ def reset_source_data():
     def create_empty_payload(fileName, projectName):
```

Comment:
if this is still only used on initialization, then lets remove the "create_empty_payload" function and call so we have one less place to worry about